specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/errored.class/response.rpt

```diff
@@ -0,0 +1,10 @@
+
+accept http://localhost:8000/snmp

```

Comment:
Please use a different endpoint (not snmp).  Suggest echo.  We probably use a convention for the paths in the other http spec tests.  Please try to follow the conventions used elsewhere.